nemo/export/quantize/quantizer.py

```diff
@@ -171,22 +157,13 @@ def dummy():
         set_tensor_parallel_group(mpu.get_tensor_model_parallel_group())
 
     @staticmethod
-    def _load_and_modify_config(
-        model_file: str,
-        tensor_model_parallel_size: Optional[int] = None,
-        pipeline_model_parallel_size: Optional[int] = None,
-    ):
-        model_cfg = load_config(model_file)
-
+    def modify_model_config(model_cfg: DictConfig) -> DictConfig:
+        """Modify model config for quantization."""
         with open_dict(model_cfg):
-            model_cfg.activations_checkpoint_method = None
-            model_cfg.activations_checkpoint_granularity = None
-            model_cfg.sequence_parallel = False
-            if tensor_model_parallel_size is not None:
-                model_cfg.tensor_model_parallel_size = tensor_model_parallel_size
-            if pipeline_model_parallel_size is not None:
-                model_cfg.pipeline_model_parallel_size = pipeline_model_parallel_size
-            # Only custom ModelOpt spec is supported for PTQ: this custom spec is largely based on local Megatron-LM
+            if model_cfg.sequence_parallel:
```

Comment:
Not sure if all models specify this - maybe for safety let's write `model_cfg.get("sequence_parallel", False)`